components/About/data.tsx

```diff
@@ -101,48 +101,43 @@ export const callsToAction = {
     {
       title: 'Hosting a node',
       content:
-        'Hosting a node using your graffiti automatically earns you points. You will only be rewarded points for hosting one node.',
-      points: ['12 hours = 100 points'],
+        'You can earn points by hosting an Iron Fish full node. To do so, make sure your telemetry is turned on, and your graffiti is set. You will only be rewarded points for hosting one node per graffiti.',
+      points: ['12 hours of continuous uptime = 100 points'],
       status: Status.New,
-      earn: 0,
-      disabled: true,
+      ctaText: 'Set up instructions',
+      href: 'https://coda.io/d/_dngSKECR9gO/Phase-2-FAQ_suBOW',
```

Comment:
This doc doesn't appear to be public yet -- is it ready to open up?

Reply:
yeah I'll change that